CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/http/apis/OpenAi.cs

```diff
@@ -256,14 +256,23 @@ public static class ChatGptExtensions {
             self.Add(new ChatGpt.Line(ChatGpt.Role.system, content: CreateJsonInstructions(exampleResponse)));
         }
 
-        public static string CreateJsonInstructions<T>(T exampleResponse) {
+        public static string CreateJsonInstructions<T>(params T[] exampleResponses) {
+            if (exampleResponses.IsNullOrEmpty()) return null;
+            
             var schemaGenerator = new ModelToJsonSchema(nullValueHandling: Newtonsoft.Json.NullValueHandling.Ignore);
             var className = typeof(T).Name;
-            JsonSchema schema = schemaGenerator.ToJsonSchema(className, exampleResponse);
-            var schemaJson = JsonWriter.GetWriter(exampleResponse).Write(schema);
-            var exampleJson = JsonWriter.GetWriter(exampleResponse).Write(exampleResponse);
+            JsonSchema schema = schemaGenerator.ToJsonSchema(className, exampleResponses[0]);
+            var schemaJson = JsonWriter.GetWriter(exampleResponses[0]).Write(schema);
             var jsonSchemaInfos = " This is the json schema that describes the format you have to use for your json response: " + schemaJson;
-            var exampleJsonInfos = " And for that schema, this would an example of a valid response: " + exampleJson;
+            var exampleJsonInfos = " ";
+            if (!exampleResponses.IsEmpty()) {
+                exampleJsonInfos += " And for that schema, these would be examples of a valid response:";
+                
+                foreach (T exampleResponse in exampleResponses) {
+                    var exampleJson = JsonWriter.GetWriter(exampleResponses).Write(exampleResponses);
+                    exampleJsonInfos += " " + exampleJson;
```

Comment:
Might it be needed to separate the examples more clearly than through a whitespace? Not sure since the json syntax probably makes it clear when one example ends and the next starts so might be fine

Reply:
The single space seems to work fine. I could not see any improvements with stronger separation.